javaparser-core/src/main/java/com/github/javaparser/printer/lexicalpreservation/Difference.java

```diff
@@ -843,14 +843,16 @@ void apply(NodeText nodeText, Node node) {
 
             if (bestMatchKey.isPresent()) {
                 correspondingIndices.add(potentialMatches.get(bestMatchKey.get()));
+            } else {
+                correspondingIndices.add(-1);
```

Comment:
With this necessary addition, some unit tests for modifiers work now aswell. This is necessary so that the code depending on this List can work in the same way as before.